Lib/os.py

```diff
@@ -704,9 +704,12 @@ def __len__(self):
         return len(self._data)
 
     def __repr__(self):
-        return 'environ({{{}}})'.format(', '.join(
-            ('{!r}: {!r}'.format(self.decodekey(key), self.decodevalue(value))
-            for key, value in self._data.items())))
+        environ_items = [
+            f"{self.decodekey(key)!r}: {self.decodevalue(value)!r}"
+            for key, value in self._data.items()
+        ]
+        formatted_items = ", ".join(environ_items)
+        return f"environ({{{formatted_items}}})"
```

Comment:
Using a generator expression is functionally equivalent with the previous version. At this point using the `environ_items` name isn't particularly useful.

```suggestion
        formatted_items = ", ".join(
            f"{self.decodekey(key)!r}: {self.decodevalue(value)!r}"
            for key, value in self._data.items()
        )
        return f"environ({{{formatted_items}}})"
```